server/routes/emailAPI.js

```diff
@@ -0,0 +1,35 @@
+const express = require("express");
+const auth = require("../middleware/auth");
+const sgMail = require("@sendgrid/mail");
+
+sgMail.setApiKey(process.env.SENDGRID_API_KEY);
+
+const router = express.Router();
+
+// Send confirmation email
+router.post("/api/email", auth, (req, res) => {
+  const msg = {
+    to: req.body.email,
+    // TODO: make project email/maybe also domain?
+    from: "uesttser@gmail.com",
```

Comment:
yeah that could be a nice idea. Probably just stick to a gmail for now, we won't provide a free domain 🤣